pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1507,6 +1507,12 @@ self: super: {
   # Fixed at head, but hasn't cut a release in awhile.
   darcs = doJailbreak super.darcs;
 
+  # Test suite's dependencies (QuickCheck etc.) are outdated
+  typerep-map = dontCheck super.typerep-map;
+
+  # Test suite is broken (uses IO while enables NoImplicitPrelude at the same time)
```

Comment:
Could you report this problem upstream, and then link to it in this comment here in the code?

We'd like to easily be able to know when we can remove these types of overrides.

Reply:
sure https://github.com/serokell/co-log-sys/issues/5 https://github.com/kowainik/typerep-map/issues/81